<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeSelector</class>
 <widget class="QFrame" name="ModeSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>76</width>
    <height>30</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>55</width>
    <height>30</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
 color: #ffffff; 
border-style: inset;
 border: 1px solid #aaaaaa;
 border-radius: 4px;
 background: #555555;
max-height: 11px;
min-height: 11px;
min-width:13px;
max-width:13px;
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QPushButton" name="push1">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>4</y>
     <width>15</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>View: Artist &gt; Album</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QLed" name="led1" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>15</y>
     <width>13</width>
     <height>4</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="push2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>4</y>
     <width>15</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>View: Year &gt; Artist &gt; Album</string>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLed" name="led2" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>15</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="push3">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>4</y>
     <width>15</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>View: Genre &gt; Artist &gt; Album</string>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="QLed" name="led3" native="true">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>15</y>
     <width>13</width>
     <height>4</height>
    </rect>
   </property>
  </widget>
  <zorder>push2</zorder>
  <zorder>push3</zorder>
  <zorder>push1</zorder>
  <zorder>led2</zorder>
  <zorder>led3</zorder>
  <zorder>led1</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLed</class>
   <extends>QWidget</extends>
   <header>qled.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
